### ISSUE TITLE
x86 disasssembly bug

### ISSUE THREAD

**Issue by DatBrick** [Vector35/binaryninja-api#1210]
Binary Ninja Version: 1.1.1442-dev, 66e96fde
Platform: Windows 10 (10.0)

There seems to be an x86 disassembler bug with instructions that read from address 0. E.G `mov rax, [0]` is shown as:
- ASM:
```
mov rax, qword []
>>> current_function.arch.get_instruction_text(bv.read(here, current_function.arch.max_instr_length), here)
(['mov', '     ', 'rax', ', ', 'qword ', '[', ']'], 8L)
```
- LLIL:
```
rax = [nop].q
>>> current_llil[0].prefix_operands
[<LLIL_SET_REG 8>, rax, <LLIL_LOAD 8>, <LLIL_NOP>]
```

**Comment by KyleMiles** [comment]
I can confirm this.  Should be fixed in 1.1.1443-dev.

**Comment by KyleMiles** [comment]
Fixed in 1.1.1443-dev.
